parmed/amber/parameters.py

```diff
@@ -211,7 +211,7 @@ def __init__(self, *filenames):
     #===================================================
 
     @classmethod
-    def from_leaprc(cls, fname):
+    def from_leaprc(cls, fname, addAtomTypes=True):
```

Comment:
I wasn't sure what you guys use here, camelCase is just strong in me from coding for OpenMM! Also that keeps the camelCase that Amber uses for this command. But anyway, this PR will not stay in this form, looks like it.